boards/Kconfig

```diff
@@ -477,6 +477,14 @@ config ARCH_BOARD_ESP32S3_DEVKIT
 		ESP32-S3-DevKitC-1 version may be based either on ESP32-S3-WROOM-1/1U or
 		ESP32-S3-WROOM-2/2U.
 
+config ARCH_BOARD_ESP32S3_8048S043
+	bool "Espressif ESP32-S3 8048S043"
```

Comment:
Espressif is not the manufactor of this board. It is manufactured by Sunton, ref: https://www.openhasp.com/0.7.0/hardware/sunton/esp32-8048s0xx/